platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/system/transaction/SystemTransactionType.java

```diff
@@ -22,11 +22,5 @@
 public enum SystemTransactionType {
 
     /** first byte of a system transaction giving a signed state's (round number, signature) */
-    SYS_TRANS_STATE_SIG,
-
-    /** first byte of a system transaction giving all avgPingMilliseconds stats (sent as ping time in microseconds) */
-    SYS_TRANS_PING_MICROSECONDS,
```

Comment:
You create as horrible big PRs as I often do :D

Reply:
😜